spk/beets/Makefile

```diff
@@ -0,0 +1,26 @@
+SPK_NAME = beets
+SPK_VERS = 1.4.3
+SPK_REV = 1
+SPK_ICON = src/beets.png
+
+WHEELS = src/requirements.txt
+SPK_DEPENDS = "python>=2.7.8-11"
+
+MAINTAINER = Volunteer?
```

Comment:
You are the maintainer, or at the very least, the creator of the package...although I hope you can keep an eye on the upstream version and update it every once in a while.